test-container/src/test/java/io/strimzi/StrimziKafkaClusterTest.java

```diff
@@ -0,0 +1,160 @@
+            kafkaClusterConfiguration);
+        systemUnderTest.start();
+
+        try (
+            AdminClient adminClient = AdminClient.create(ImmutableMap.of(
+                AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, systemUnderTest.getBootstrapServers()
+            ));
+
+            KafkaProducer<String, String> producer = new KafkaProducer<>(
+                ImmutableMap.of(
+                    ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, systemUnderTest.getBootstrapServers(),
+                    ProducerConfig.CLIENT_ID_CONFIG, UUID.randomUUID().toString()
+                ),
+                new StringSerializer(),
+                new StringSerializer()
+            );
+
+            KafkaConsumer<String, String> consumer = new KafkaConsumer<>(
+                ImmutableMap.of(
+                    ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, systemUnderTest.getBootstrapServers(),
+                    ConsumerConfig.GROUP_ID_CONFIG, "tc-" + UUID.randomUUID(),
+                    ConsumerConfig.AUTO_OFFSET_RESET_CONFIG,  OffsetResetStrategy.EARLIEST.name().toLowerCase(Locale.ROOT)
+                ),
+                new StringDeserializer(),
+                new StringDeserializer()
+            );
+        ) {
+            final String topicName = "example-topic";
+            final String recordKey = "strimzi";
+            final String recordValue = "the-best-project-in-the-world";
+
+            Collection<NewTopic> topics = Collections.singletonList(new NewTopic(topicName, numberOfReplicas, (short) numberOfReplicas));
+            adminClient.createTopics(topics).all().get(30, TimeUnit.SECONDS);
+
+            consumer.subscribe(Collections.singletonList(topicName));
+
+            producer.send(new ProducerRecord<>(topicName, recordKey, recordValue)).get();
+
+            TestUtils.waitFor("Consumer records are present", Duration.ofSeconds(10).toMillis(), Duration.ofMinutes(1).toMillis(),
+                () -> {
+                    ConsumerRecords<String, String> records = consumer.poll(Duration.ofMillis(100));
+
+                    if (records.isEmpty()) {
+                        return false;
+                    }
+
+                    // verify count
+                    assertThat(records.count(), is(1));
+
+                    ConsumerRecord consumerRecord = records.records(topicName).iterator().next();
+
+                    // verify content of the record
+                    assertThat(consumerRecord.topic(), is(topicName));
+                    assertThat(consumerRecord.key(), is(recordKey));
+                    assertThat(consumerRecord.value(), is(recordValue));
+
+                    return true;
+                });
+
+            consumer.unsubscribe();
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        } catch (ExecutionException e) {
+            e.printStackTrace();
+        } catch (TimeoutException e) {
+            e.printStackTrace();
```

Comment:
Calling `printStackTrace` is almost never the right thing to do, it's usually a sign that exception handling isn't really being done properly. In this case, because it's a test, just declare `throws Exception`.